runtime/src/main/kotlin/kotlin/ranges/Ranges.kt

```diff
@@ -82,40 +82,3 @@ public class LongRange(start: Long, endInclusive: Long) : LongProgression(start,
         public val EMPTY: LongRange = LongRange(1, 0)
     }
 }
-
-/**
- * A closed range of values of type `Float`.
- *
- * Numbers are compared with the ends of this range according to IEEE-754.
- */
-private class ClosedFloatRange (
-        start: Float,
-        endInclusive: Float
-): ClosedFloatingPointRange<Float> {
```

Comment:
FYI: Moved to common https://upsource.jetbrains.com/kotlin/revision/19e001afad2bfadb987d5a5ee807e11caf1c8e19